playbooks/dependencies.md

```diff
@@ -0,0 +1,91 @@
+  framework )_ every line of code you add is code we need to maintain, which again can make migrations hard ( The
+  code we don’t use may be blocking an upgrade )
+
+- More dependencies means more code churn _( Their APIs may change across versions, and if you don’t keep
+  up-to-date then larger jumps are harder. They may have security vulnerabilities which means you have to update
+  that dependencies and get any transitives in sync )_
+
+## A rubric on whether a dependency needs an RFC?
+
+If you answer yes to most of these questions, then you probably don’t need to write an RFC
+
+- Could you fit this codebase in your head after reading the source?
+- Is this the stand-out obvious answer to a particular domain problem?
+- Do you expect your team to be the only people who know about this dependency?
+- Is this obviously being used in production by the maintainers, and thus battle-tested?
+- Are the dependencies of this dependency already in our projects, or is the dependency itself a transitive
+  dependency of another dependency we already rely on?
+- Would this dependency be the first time we’ve needed something of this domain?
+- Do you feel well versed in the domain of this dependency and/or could maintain it if that needs to become an
+  option?
+
+## Ways to safely work around this
+
+- Re-use existing dependencies, or build the exact implementation that addresses just our needs today. Rather than
+  trying to foresee the future, we should re-asses once we have more information.
+
+- Directly import just the code you need, see [Importing directly](#importing-directly).
+
+## Importing Directly
+
+If you need a single function or file of a dependency that is BSD, MIT, or ISC licensed; you can directly copy the
+source code into your app (i.e. [vendor][] it). This code **should** contain a link to the original code and
+**must** need a reference to the original license; which can either be a LICENSE file in the same folder, or the
+inclusion of a stable license URL (such as those hosted by the OSI) in a comment at the top.
+
+When doing so, be sure to call-out the code that’s being vendored. Because of legal reasons, we don’t want the
+code-reviewer to accidentally gloss over it.
+
+## Notes
+
+Prior Artsy engineer’s writing on the topic of dependency management:
+
+- https://github.com/artsy/README/issues/117
+- http://artsy.github.io/blog/2015/09/18/Cocoa-Architecture-Dependencies/
+- https://www.objc.io/issues/22-scale/artsy/
+- https://www.fngtps.com/2013/a-quick-note-on-minimal-dependencies-in-ruby-on-rails/
+- http://artsy.github.io/blog/2017/01/04/OSS-FAQ/
+
+Referential time-sinks, but valuable for extra info
+
+- https://guides.cocoapods.org/making/quality-indexes.html
+- https://gist.github.com/dominictarr/9fd9c1024c94592bc7268d36b8d83b3a
+- https://blog.npmjs.org/post/141577284765/kik-left-pad-and-npm
+
+[vendor]: https://en.wiktionary.org/wiki/vendor#Verb
```

Comment:
We should link to something that explains the usage of "vendor" specific to dependency management. I can't find a whole lot, but maybe something like [this](https://codeengineered.com/blog/2015/go-should-i-vendor/) would work.